src/java/org/apache/cassandra/tools/NodeTool.java

```diff
@@ -262,6 +263,13 @@ public int execute(String... args)
                .withCommand(RepairAdmin.SummarizePendingCmd.class)
                .withCommand(RepairAdmin.SummarizeRepairedCmd.class);
 
+        builder.withGroup(CMSCmdGroupHelp.CMS_CMD_GROUP_NAME)
```

Comment:
I am not sure if we do not want to put all classes under "CMSAdmin" or something similar as it is done in `RepairAdmin` case but I am overall OK with this as well. I can not estimate if putting it all into one class is better than leaving it like this.